javascript/ql/src/experimental/Security/CWE-347/JWTMissingSecretOrPublicKeyVerification.ql

```diff
@@ -0,0 +1,19 @@
+/**
+ * @name JWT Missing Secret Or Public Key Verification
+ * @description The software does not verify the JWT token with a cryptographic secret or public key.
+ * @kind problem
+ * @problem.severity warning
+ * @precision high
+ * @id js/jwt-missing-secret-or-public-key-verification
+ * @tags security
+ *       external/cwe/cwe-347
```

Comment:
I'm not sure if `cwe-347` is the best fit, how about [`cwe-295`](https://cwe.mitre.org/data/definitions/295.html) (Improper Certificate Validation). 

(You could also just add `cwe-295` to the list if you think `cwe-347` is a better fit).

Reply:
I still prefer CWE-347, according to its definition, it fits perfectly the exploit.
_The software does not verify, or incorrectly verifies, the cryptographic signature for data._

CWE-295 seems more appropriate for certificates.